docs/gen-ai/gen-ai-spans.md

```diff
@@ -1,8 +1,8 @@
 <!--- Hugo front matter used to generate the website version of this page:
-linkTitle: Generative AI traces
+linkTitle: GenAI spans
```

Comment:
Above we have a folder named "database" and then this link is just "Spans". Why is it different here? 🤔 .

Reply:
Thanks for noticing the inconsistency. In this section, I'll fix that.